### ISSUE TITLE
How to get the phoneme?

### ISSUE THREAD

**Issue by psuu0001** [jonatasgrosman/wav2vec2-sprint#5]
Hi, bro, very appreciate for your work. The method to load the model can directly get the word. But I want the phoneme. Do you know how to get the phoneme rather than word?

**Comment by jonatasgrosman** [comment]
Hi @psuu0001 sorry for the very late reply. I had some health issues that forced me to stay offline for a while. My life is going back to normal slowly now...

I'll try to help you here, but I think at this time you probably already know the answer...

The fine-tuned model will only handle the labels you trained it on. So if you want the model to output phonemes, you'll need to train it using phonemes as training labels.
Unfortunately, all my models have been trained on letters, not phonemes, so they don't output phonemes, just letters. So if you need to have phonemes as outputs, you'll need to fine-tune a new model using phonemes as training labels, or you can post-process my models' output using some tool for word->phoneme conversion like https://github.com/bootphon/phonemizer

P.S.: This repository is now deprecated in favor of https://github.com/jonatasgrosman/huggingsound

**Comment by kerolos** [comment]
it means that we have to train the model from scratch to get a phonemes based model. My question, how we can put the lexicon in the model to be used during training , Is it also can handle if a word has many transcription ?
Is there any way to transfer learning the existing model and change the last layer from letter based to phoneme based and put it with high learning rate?
